docs/topics/test-system.md

```diff
@@ -1,72 +1,126 @@
+  - https://wikibase-qs-testing-previous.wmcloud.org (8282 internal)
 
-```
-someone@wb-product-testing:/opt/wmde.2$ tree
-.
-├── example-fedprops
-│   ├── docker-compose.extra.yml
-│   ├── docker-compose.yml
-│   ├── extra-install.sh
-│   ├── local.env
-│   ├── LocalSettings.php
-│   ├── README.md
-│   └── template.env
-└── example-full
-    ├── docker-compose.extra.yml
-    ├── docker-compose.yml
-    ├── extra-install.sh
-    ├── LocalSettings.php
-    ├── README.md
-    └── template.env
-
-2 directories, 13 files
+These proxies are configured in Horizon https://horizon.wikimedia.org/project/proxy/
 
-```
+All of the internal ports need to be opened in the firewall for traffic from the proxies to make it to the docker hosted ports https://horizon.wikimedia.org/project/security_groups/3d459ac5-e78a-4a5e-a5e0-d31176ed3628/
 
-## Update with new release
+## Initial creation
 
+On an empty machine first you need some dependencies:
 
-Normally `example-full` is the environment you want to use. The folder should contain an `.env` file that can be updated with the new image versions produced by the [release-pipeline](https://github.com/orgs/wmde/packages?repo_name=wikibase-release-pipeline).
+- Follow the `docker` install docs at https://docs.docker.com/engine/install/debian/
+- Follow the `docker-compose` install docs at https://docs.docker.com/compose/install/
 
-### 1. stop the containers 
+You can create the directory to store the test systems:
 
 ```sh
-docker-compose -f docker-compose.yml -f docker-compose.extra.yml down
+sudo mkdir /opt/test-systems
+sudo chown mediawiki:wikidev /opt/test-systems
+sudo chmod +775 /opt/test-systems
 ```
 
-### 2. Update image version in .env file
+You can create a test system using the parameterized bash code below.
+The inputs are:
 
-Example for workflow run [1157808966](https://github.com/wmde/wikibase-release-pipeline/actions/runs/1157808966).
+ - TEST_SYSTEM: The name of the test system to create, should be either "latest" or "previous"
+ - EXAMPLE_HASH: Hash of the release pipeline repository to use the docker-compose example from
+ - BUILD_NUMBER: Build of images, or tag, to use for images of the test system
 
-```
-WIKIBASE_IMAGE_NAME=ghcr.io/wmde/wikibase:1157808966
-WDQS_IMAGE_NAME=ghcr.io/wmde/wdqs:1157808966
-WDQS_FRONTEND_IMAGE_NAME=ghcr.io/wmde/wdqs-frontend:1157808966
-ELASTICSEARCH_IMAGE_NAME=ghcr.io/wmde/elasticsearch:1157808966
-WIKIBASE_BUNDLE_IMAGE_NAME=ghcr.io/wmde/wikibase-bundle:1157808966
-QUICKSTATEMENTS_IMAGE_NAME=ghcr.io/wmde/quickstatements:1157808966
-WDQS_PROXY_IMAGE_NAME=ghcr.io/wmde/wdqs-proxy:1157808966
-MYSQL_IMAGE_NAME=mariadb:10.3
-```
-
-### 3. start containers again 
+Do the following (with the parameters you require)...
 
 ```sh
-docker-compose -f docker-compose.yml -f docker-compose.extra.yml up -d
+# Inputs for setup
+TEST_SYSTEM=latest
+EXAMPLE_HASH=b8aa96cb0cd99054631b558535ef1f3a9b8d41b8
+BUILD_NUMBER=1824280943
+
+TEST_SYSTEM=previous
```

Comment:
Would it make sense to comment one out by default?

Reply:
Yes! this was actually a mistake as I was using the docs as a dumping place for the vars that I was putting in when setting up the 2 systems!

Reply:
https://github.com/wmde/wikibase-release-pipeline/pull/315/commits/e5583541482bce62201ddc1d5d0a67a288e2ed92